packages/reference/src/components/MissingEntityCard/MissingEntityCard.tsx

```diff
@@ -13,23 +14,24 @@ export function MissingEntityCard(props: {
 }) {
   return (
     <Card className={styles.card}>
-      <div className={props.asSquare ? styles.squareCard : ''}>
-        <SectionHeading marginBottom="none">
-          {props.entityType === 'Entry' && 'Entry is missing or inaccessible'}
-          {props.entityType === 'Asset' && 'Asset is missing or inaccessible'}
-        </SectionHeading>
-      </div>
-      {!props.isDisabled && props.onRemove && (
-        <IconButton
-          variant="transparent"
-          icon={<CloseIcon variant="muted" />}
-          className={styles.close}
-          aria-label="Delete"
-          onClick={() => {
-            props.onRemove && props.onRemove();
-          }}
-        />
-      )}
+      <Flex alignItems="center" justifyContent="space-between">
+        <div className={props.asSquare ? styles.squareCard : ''}>
+          <SectionHeading marginBottom="none">
+            {props.entityType === 'Entry' && 'Entry is missing or inaccessible'}
+            {props.entityType === 'Asset' && 'Asset is missing or inaccessible'}
+          </SectionHeading>
```

Comment:
I wonder if it can be like

```suggestion
          <SectionHeading marginBottom="none">
            {`{props.entityType} is missing or inaccessible`}
          </SectionHeading>
```